libr/core/cmd_print.c

```diff
@@ -1629,6 +1629,7 @@ static void disasm_strings(RCore *core, const char *input, RAnalFunction *fcn) {
 	free (string2);
 	free (string);
 	free (s);
+	free (str);
```

Comment:
mm i don't see why after that, the thing is that str is only allocated at (at least i am aware of)

```
		if (str) {
			char *qoe = strstr (str, ";");
			if (qoe) {
				//XXX str leaks
				str = r_str_ndup (str, qoe - str);
			}
		}

```

but the rest of cases is just a pointer returned by strstr